domino-ui/src/main/java/org/dominokit/domino/ui/popover/Popover.java

```diff
@@ -137,7 +137,7 @@ public static Popover createPicker(HTMLElement target, Node content) {
         return popover;
     }
 
-    public static Popover createPicker(IsElement target, IsElement content) {
+    public static <E1 extends HTMLElement, E2 extends HTMLElement> Popover createPicker(IsElement<E1> target, IsElement<E2> content) {
```

Comment:
`public static Popover createPicker(IsElement<?> target, IsElement<?> content) {`
doing it like this should be enough